content/en/docs/reference/config/analysis/ist0105/index.md

```diff
@@ -3,7 +3,7 @@ title: IstioProxyVersionMismatch
 layout: analysis-message
 ---
 
-This message is emitted regarding a pod when:
+This message occurs regarding a pod when:
 
 * Automatic sidecar injection is enabled (default enabled unless explicitly
   disabled via the helm template variable `sidecarInjectorWebhook.enabled`)
```

Comment:
We've purged helm stuff out of the doc set. @frankbu can provide the replacement text to use here.

Reply:
fixed